setup.py

```diff
@@ -37,6 +37,7 @@
             'lxml',
             'mock',
             'plone.app.robotframework>0.9.16',
+            'robotframework-debuglibrary',
```

Comment:
@tisto It's not a default dependency. "Debug" keyword in p.a.robotframework is a bit fancy. It is dependency with [debug] extras, but that can only be defined in buildout.

1. open debuglibrary console when that's available
2. pause with dialog when that dialog is supported by python (it's optional Python build dep)
3. as last fallback stops with pdb

That said, it's not a perfect dependency. A better option would have been to put it somewhere into buildout.coredev so that's its included in modules of bin/test, but we didn't immediately find the correct testrunner recipe option for that.